### ISSUE TITLE
Cookie Banner (Privacy) Looks Ugly on Mobile

### ISSUE THREAD

**Issue by jwlim94** [tcampb/react-calendly#193]
I embedded the Calendly to our web page. The cookie banner looks ugly in mobile view. Is there a way to make this better such as showing this cookie banner in a bottom drawer or something?

![Screenshot 2024-12-20 at 5 04 18 PM](https://github.com/user-attachments/assets/8fb97cf3-11d9-4107-bac5-70d6c861173e)

**Comment by tcampb** [comment]
Hey @jwlim94 - 

I would recommend using the `hideGdprBanner` property to disable Calendly's default cookie banner. This would allow you to show your own cookie banner on the site.

```javascript
import React from "react";
import { InlineWidget } from "react-calendly";

const App = () => {
  return (
    <div className="App">
      <InlineWidget url="https://calendly.com/your_scheduling_page" pageSettings={ { hideGdprBanner: true } } />
    </div>
  );
};

export default App;
```

**Comment by jwlim94** [comment]
@tcampb As we use `hideGdprBanner`, is it required to show our own cookie banner on the site?

**Comment by tcampb** [comment]
Not necessarily, it would depend on whether or not GDPR applies to your website.
